auth_session_timeout/models/ir_config_parameter.py

```diff
@@ -13,24 +12,18 @@
 class IrConfigParameter(models.Model):
     _inherit = 'ir.config_parameter'
 
-    @tools.ormcache(skiparg=0)
-    def get_session_parameters(self, db):
-        param_model = self.pool['ir.config_parameter']
-        cr = self.pool.cursor()
-        delay = False
-        urls = []
-        try:
-            delay = int(param_model.get_param(
-                cr, SUPERUSER_ID, DELAY_KEY, 7200))
-            urls = param_model.get_param(
-                cr, SUPERUSER_ID, IGNORED_PATH_KEY, '').split(',')
-        finally:
-            cr.close()
-        return delay, urls
+    @api.model
+    @tools.ormcache('self.env.db')
```

Comment:
Thanks for the excellent explanation @Yajo - this totally clarifies what we are going for here & I think points me in the right direction. I'll make the update soon 😄